sched/init/nx_start.c

```diff
@@ -588,6 +589,12 @@ void nx_start(void)
   nxmsg_initialize();
 #endif
 
+#ifdef CONFIG_FS_SHMFS
+  /* Initialize posix shm support */
+
+  shmfs_initialize();
```

Comment:
move to fs_initialize

Reply:
Done, thanks for pointing this out. Also cleaned away now unnecessary header files.